Automattic/wp-calypso
# Copy Site: Prototype flow to copy an existing site

- Closes https://github.com/Automattic/dotcom-forge/issues/1468

#### Proposed Changes

* Add a new `Copy Site` item on SMP site actions for atomic sites that you own.
* Create a new rough `Copy Site` flow to create a new site, buy a plan and migrate it to Atomic.

#### Testing Instructions


* If It's not yet deployed, Apply and sandbox D97450-code.
* Go to `/sites`.
* On an atomic site you own, Click on the ellipsis action menu.
* Click on `Create a copy of this site`.
* Observe a new flow is open.
* Follow the steps, pay the plan, and observe a Success message after copying your site.




#### Pre-merge Checklist
